pandas/tests/test_algos.py

```diff
@@ -355,19 +371,23 @@ def test_factorize_na_sentinel(self, sort, na_sentinel, data, uniques):
         else:
             tm.assert_extension_array_equal(uniques, expected_uniques)
 
-
 class TestUnique:
```

Comment:
Note that no tests are removed here (even though the diff is large). I sometimes joined tests and parametrized them with fixtures. In fact, there should be many more tests now...